en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown

```diff
@@ -108,3 +104,89 @@ procedure takes only a few steps:
+
+Here we provide an example usage of Elasticsearch's [snapshot and restore](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html) feature to back-up and restore _Search Tuning_ indexes and documents.
+
+1. Create a folder called `elasticsearch_local_backup` somewhere in your system where Elasticsearch has read and write access to, for example `/path/to/elasticsearch_local_backup`
+
+2. Edit the `elasticsearch.yml` on [all master and data nodes](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#snapshots-filesystem-repository) in your Elasticsearch cluster and add
+
+    ```yaml
+    path.repo: [ "/path/to/elasticsearch_local_backup" ]
+    ```
+
+3. Restart your Elasticsearch nodes
+
+4. [Register the snapshot repository](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html): run the following API request (for example through the Dev Tools console in Kibana):
+
+    ```
+    PUT /_snapshot/elasticsearch_local_backup
+    {
+      "type": "fs",
+      "settings": {
+        "location": "/path/to/elasticsearch_local_backup"
+      }
+    }
+    ```
+
+5. [Create a snapshot](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-take-snapshot.html):
+
+    ```
+    PUT /_snapshot/elasticsearch_local_backup/snapshot1?wait_for_completion=true
+    {
+      "indices": "liferay-search-tuning*",
+      "ignore_unavailable": true,
+      "include_global_state": false
+    }
+    ```
+
+    If you want to create a snapshot for all Liferay indexes, you can use `"indices": "liferay*,workflow-metrics*"` instead.
+
+6. To [restore](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-restore-snapshot.html) only specific indexes from a snapshot under a different name, run an API call similar to this:
+
+    ```
+    POST /_snapshot/elasticsearch_local_backup/snapshot1/_restore
+    {
+      "indices": "liferay-20101-search-tuning-synonyms,liferay-20101-search-tuning-rankings",
+      "ignore_unavailable": true,
+      "include_global_state": false,
+      "rename_pattern": "(.+)",
+      "rename_replacement": "restored_$1",
+      "include_aliases": false
+    }
+    ```
+
+    where `indicies` sets the index names to restore from (as they are stored in the snapshot!). The indexes from the above call will be restored as `restored_liferay-20101-search-tuning-rankings` and `liferay-20101-search-tuning-synonyms` according to the `rename_pattern` and `rename_replacement` regular expressions.
+
+Restoring the _Search Tuning_ indexes from a snapshot under different names comes in handy, if you did not upgrade your Elasticsearch 6 cluster, but rather, you just set-up your new Elasticsearch 7 cluster, configured Elasticsearch 7 in @product-ver@ and performed a full reindex from @product@.
```

Comment:
@lipusz, sorry for my ignorance, but I'm not sure I understand. What happens if you didn't create the backups with a new name?

Reply:
Maybe a better questions is, why would you ever create the backups with the same name?

Reply:
@rbohl Which part is not clear?

Reply:
> Restoring the _Search Tuning_ indexes from a snapshot **under different names comes in handy, if you did not upgrade your Elasticsearch 6 cluster, but rather, you just set-up your new Elasticsearch 7 cluster**, configured Elasticsearch 7 in @product-ver@ and performed a full reindex from @product@.

Reply:
I think I phrased it like that for safety reasons: of course you could restore with the original name too.

Reply:
Yeah, but I need to understand the reason for this better so I can clarify the text :wink:

Reply:
I want to say something like

> Creating the search tuning backups using new index names (e.g., prepended with `restored_`), then restoring them to the cluster using the original index name is a best practice. This is because in certain circumstances,...

I don't understand what to put in the `...`